snapcraft/plugins/kernel.py

```diff
@@ -23,8 +23,8 @@
 The following kernel specific options are provided by this plugin:
 
     - kernel-image-target:
-      (string; default: bzImage)
-      the kernel image make target to build; maps to make target.
+      (yaml object; default: bzImage)
```

Comment:
Shouldn't this be: a string or a yaml object?
I think the help should explain that you can pass only one target, and how.